### DISCUSSION TITLE
[feature] 建议更换华为云DNS的默认endpoint

### DISCUSSION THREAD

**Discussion by HanadaLee** [NewFuture/DDNS#266] [Ideas]
目前ddns内部使用的endpoint为dns.myhuaweicloud.com，位于国内，被墙的服务器将会无法调用API完成DNS记录更新，建议换成位于香港的dns.ap-southeast-1.myhuaweicloud.com，可以将兼顾大陆和海外的请求。

**Comment by NewFuture** [comment]
这个需要大家讨论一下

**Reply by HanadaLee** [reply]
> 这个需要大家讨论一下

好像“讨论”一年了都没结果

**Comment by NewFuture** [comment]
#504
